account_product_fiscal_classification/views/view_product_category.xml

```diff
@@ -7,41 +7,41 @@
-                            string="Apply to Child Categories" icon="gtk-dnd-multiple" colspan="2"/>
-                    <field name="fiscal_classification_ids" widget="many2many_tags" colspan="4"
-                            attrs="{'invisible': [('fiscal_restriction', '=', False)]}"/>
-                    <group name="forbidden" string="Products with Forbidden Classification" colspan="4" col="4"
-                            attrs="{'invisible': [('forbidden_classification_template_qty','=', 0)]}">
-                        <field name="forbidden_classification_template_qty" colspan="2"/>
-                        <newline />
-                        <field name="forbidden_classification_template_ids" nolabel="1" colspan="4">
-                            <tree>
-                                <field name="name"/>
-                                <field name="fiscal_classification_id"/>
-                            </tree>
-                        </field>
-                    </group>
-                </group>
-            </xpath>
-        </field>
-    </record>
-
-    <record id="view_product_category_tree" model="ir.ui.view">
-        <field name="model">product.category</field>
-        <field name="inherit_id" ref="product.product_category_list_view"/>
-        <field name="arch" type="xml">
-            <field name="complete_name" position="after">
-                <field name="fiscal_classification_ids" widget="many2many_tags"/>
+  <record id="view_product_category_form" model="ir.ui.view">
+    <field name="model">product.category</field>
+    <field name="inherit_id" ref="account.view_category_property_form"/>
+    <field name="arch" type="xml">
+      <xpath expr="//group[@name='account_property']" position="after">
+        <group name="fiscal_property" string="Fiscal Settings" colspan="4" col="4">
+          <field name="fiscal_restriction"/>
+          <button name="apply_classification_to_childs" type="object"
+            string="Apply to Child Categories" icon="gtk-dnd-multiple" colspan="2"/>
+          <field name="fiscal_classification_ids" widget="many2many_tags" colspan="4"
+            attrs="{'invisible': [('fiscal_restriction', '=', False)]}"/>
+          <group name="forbidden" string="Products with Forbidden Classification" colspan="4" col="4"
+            attrs="{'invisible': [('forbidden_classification_template_qty','=', 0)]}">
+            <field name="forbidden_classification_template_qty" colspan="2"/>
+            <newline/>
+            <field name="forbidden_classification_template_ids" nolabel="1" colspan="4">
+              <tree>
+                <field name="name"/>
+                <field name="fiscal_classification_id"/>
+              </tree>
             </field>
-        </field>
-    </record>
+          </group>
+        </group>
+      </xpath>
+    </field>
+  </record>
+
+  <record id="view_product_category_tree" model="ir.ui.view">
```

Comment:
Nice improvement, could you add a group to this view. (like accounting manager). I'm not sure "basic" users are interested by this information.

Reply:
Done

Reply:
![classif](https://user-images.githubusercontent.com/1853434/36205228-514cccd4-118e-11e8-9f25-1a6d8c42c28f.png)